solar/solar/core/signals.py

```diff
@@ -16,7 +16,7 @@
 import networkx
 
 from solar.core.log import log
-from solar.interfaces import orm
+# from solar.interfaces import orm
```

Comment:
We need to remove "all", I sometimes leave this just in case, if we need some old reference etc.